src/logger.py

```diff
@@ -2,7 +2,8 @@
 
 
 def create_logger(name: str, level: str, format: str) -> logging.Logger:
-    logging.basicConfig(format=format)
+    # To be cancell, just to log seleniumwire calls
+    logging.basicConfig(format=format, level="INFO")
```

Comment:
```suggestion
    logging.basicConfig(format=format, level=level)
```
Now we can set the LOG_LEVEL environment variable with the value INFO